examples/fips/README.md

```diff
@@ -0,0 +1,85 @@
+# FIPS Compliance
+
+We tested FIPS compliance for the client using OpenSSL 3.0. To use the client in FIPS-compliant mode, use OpenSSL 3.0. Older versions of OpenSSL have not been verified (although they may work).
+
+## Communication between client and Kafka cluster
+
+### Install from prebuilt wheels
+
+If you install the this client through prebuilt wheels using `pip`, it comes with librdkafka in which OpenSSL 3.0 is already statically linked. To enable this client to communicate with the Kafka cluster using the OpenSSL FIPS provider and FIPS-approved algorithms, you must enable the FIPS provider. You can find steps to enable the FIPS provider in section [Enabling FIPS provider](#enabling-fips-provider).
+
+
+### Install from source by building librdkafka
+
+When you build the python client from source, librdkafka dynamically links to the OpenSSL present in the system. If the installed OpenSSL is already working in FIPS mode, then you can directly jump to section [Client configuration to enable FIPS provider](#client-configuration-to-enable-fips-provider) to enable `fips` provider, otherwise use steps mentioned in section [Enabling FIPS provider](#enabling-fips-provider) to make OpenSSL in your system FIPS compliant and then enable `fips` provider. Once that is done, librdkafka and python client will use FIPS approved algorithms for the communication between client and Kafka Cluster using producer, consumer or admin client.
+
+### Enabling FIPS provider
+
+To enable the FIPS provider, you must have the FIPS module available on your system, plug the module into OpenSSL, and then configure OpenSSL to use the module. 
+You can plug the FIPS provider into OpenSSL two ways: 1) put the module in the default module folder of OpenSSL or 2) point to the module with the environment variable, `OPENSSL_MODULES`. For example: `OPENSSL_MODULES="/path/to/fips/module/lib/folder/` 
```

Comment:
You could specify that 1 can be used only when installing from source, as with the prebuild wheel it will search a path that corresponds to the Semaphore OpenSSL source folder. With the prebuild wheel only option 2 can be used, if I'm correct.